docs/docs/how-to/cloud/adding-a-custom-domain.md

```diff
@@ -0,0 +1,100 @@
+
+ 
+
+## 2. Configuring Your Domain Name System (DNS)
+Next you'll need to visit your 3rd party DNS registrar (e.g. GoDaddy, NameCheap, etc.) to add some DNS records. A DNS Record is an entry in a DNS with details about how to route a request for a given domain name.
+
+ 
+
+### Apex Domain Configuration
+If you added an apex domain, you will need to add an Address Mapping Record  (A Record). 
+
+ 
+
+Copy **one of the IP addresses** from the **value** column for the **type A** records shown next to your custom domain. In your DNS provider's domain settings, create an A Record for the IP address you chose.
+
+![Copy IP Address for A Record](../../images/mceclip1.png)
+
+The A Record **must** point to one of the **IP addresses** shown for your site. It is only necessary to create an A Record for **one** of the IP addresses listed.
+
+ 
+
+### Subdomain Configuration
+If you added a subdomain, you will need to add a **Canonical Name Record (CNAME)**. The CNAME record **must** point to the **default domain** for your site.
+
+![Configure a subdomain](../../images/mceclip2.png)
+
+ 
+
+### How to Add DNS Records for Popular Registrars
+Here's documentation on adding DNS records for some popular name registrars:
+
+- Go Daddy: [A Record](https://www.godaddy.com/help/add-an-a-record-19238) | [CNAME](https://www.godaddy.com/help/add-a-cname-record-19236)
+- Name.com: [A Record](https://www.namecheap.com/support/knowledgebase/article.aspx/319/2237/how-can-i-set-up-an-a-address-record-for-my-domain/) | [CNAME](https://www.namecheap.com/support/knowledgebase/article.aspx/9646/2237/how-to-create-a-cname-record-for-your-domain/)
+- NameCheap: [A Record](https://www.name.com/support/articles/115004893508-Adding-an-A-record) | [CNAME](https://www.name.com/support/articles/115004895548-Adding-a-CNAME-Record)
+- Google Domains: [A Record](https://support.google.com/a/answer/2579934?hl=en&ref_topic=2721296) | [CNAME](https://support.google.com/a/answer/47283)
+## 3. Check DNS Status
+Once you've added your DNS record, click **Check Status** or refresh the page to check that your DNS record has been recognized by Gatsby Hosting.
+
+![Check DNS Status](../../images/mceclip3.png)
+
+ 
+
+If your domain fails to validate after 24 hours, try creating an A Record for one of the other IP addresses available instead.
+
+![Domain failed to validate](../../images/mceclip2.png)
+
+ 
+
+## 4. Generate SSL Certificate
+After the domain is validated, we'll generate a free SSL certificate via Let's Encrypt and automatically configure it. This whole process can take a few minutes to complete.
+
+![Generating SSL Certificate](../../images/mceclip4.png)
+
+**NOTE**: DNS records need to be fully propagated in order to properly verify the certificates. If not, SSL certification generation will fail. 
```

Comment:
```suggestion
**Please Note**: DNS records need to be fully propagated in order to properly verify the certificates. If not, SSL certification generation will fail. 
```

All caps is bad for screen reader users, we normally use "Please Note"

Reply:
Lowercase `note`? (<3 that PR review, Lennart)